### ISSUE TITLE
The app repeats the entire login process whenever we re-open it after closing .

### ISSUE THREAD

**Issue by solo-daemon** [Bugheist/Flutter#204]
/assign

**Comment by letsintegreat** [comment]
As I understand access token is being stored locally, and is not fetched every time we reopen the app. Then what does it mean for the app to repeat the entire login process? What is "login process"?

**Comment by solo-daemon** [comment]
https://user-images.githubusercontent.com/65587505/222790092-b4c21f0e-242c-45ac-9299-43b5e6c019ca.mp4

**Comment by letsintegreat** [comment]
@solo-daemon That's the expected behaviour as you didn't opt to "remember me" option.

**Comment by solo-daemon** [comment]
Hmm you are right @letsintegreat i will be changin the name of the issue .

https://user-images.githubusercontent.com/65587505/222796932-1c5777e8-6906-4df8-9b4f-b5d1081a7024.mp4

 am changing the name of this issue

**Comment by letsintegreat** [comment]
@solo-daemon That is also expected. If user pops the issues page by pressing the back button, they are supposed to be logged out by design. This behaviour is defined in `WillPop` of the issues page.

**Comment by solo-daemon** [comment]
@letsintegreat I think that should not happen ideally the user should be prompted at least once before leaving the homepage and should only be forgotten by the app after he/she has pressed logout .

**Comment by letsintegreat** [comment]
@solo-daemon Agreed, a confirmation dialog should be shown when the user presses back. If confirmed logout, else back to homepage.

**Comment by solo-daemon** [comment]
Yep! exactly .

**Comment by solo-daemon** [comment]
@fredfalcon @just-ary27 Should I proceed with this?
